torch/fx/experimental/symbolic_shapes.py

```diff
@@ -3040,12 +3041,17 @@ def hint(s):
                 sources_tensors_constraints = [(source, t, context.constraint_sizes)]
 
             for src, curr_t, constraint in sources_tensors_constraints:
-                for i, ss in enumerate(curr_t.size()):
-                    property_source = TensorPropertySource(src, TensorProperty.SIZE, i)
-                    track_symint(property_source, ss, constraint[i])
-                for i, ss in enumerate(curr_t.stride()):
-                    track_symint(TensorPropertySource(src, TensorProperty.STRIDE, i), ss)
-                track_symint(TensorPropertySource(src, TensorProperty.STORAGE_OFFSET), curr_t.storage_offset())
+                if is_sparse_any(curr_t):
+                    for i, ss in enumerate(curr_t.size()):
+                        property_source = TensorPropertySource(src, TensorProperty.SIZE, i)
+                        track_symint(property_source, ss, constraint[i])
```

Comment:
I feel there are some integers we have to track, like nnz?

Reply:
What is the exact function of tracking sparse tensor shape values and other integer parameters? Namely, a sparse tensor can be viewed as a tuple of indices and values tensors. While the sparse tensor shape and nnz is related to the shapes of indices and values, should we also track the strides of indices and values tensors (we do support non-contiguous values, for instance)?